package.json

```diff
@@ -79,16 +79,17 @@
     "xml-js": "^1.6.11"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "start:server": "node build/server.js",
+    "start": "TZ=Europe/Helsinki react-scripts start",
+    "start:server": "TZ=Europe/Helsinki node build/server.js",
```

Comment:
are these env variables required here?

Reply:
While our docker host has UTC-timezone, this seems to be only way to set the timezone for the node server: https://stackoverflow.com/questions/8083410/how-can-i-set-the-default-timezone-in-node-js
Other question is that do we need that anyway, since server is only used for ssr?

Reply:
I think this should not be needed

Reply:
This can be a problem at some point if we format or parse timestamps on ssr: https://stackoverflow.com/questions/24655522/should-the-server-or-the-client-handle-timezone-when-sending-receiving-dates
One option is to use only epoch time (milliseconds from 1970) and then render format on client side on user's timezone.
This might be a good idea especially with internet events where user can join anywhere in the world. On the other hand many users might have strong assumption that physical event times are shown in helsinki timezone time.
I can remove this as it is not a problem for now.